homeassistant/components/islamic_prayer_times/const.py

```diff
@@ -7,6 +7,22 @@
 NAME: Final = "Islamic Prayer Times"
 
 CONF_CALC_METHOD: Final = "calculation_method"
-
 CALC_METHODS: list[str] = list(PrayerTimesCalculator.CALCULATION_METHODS)
 DEFAULT_CALC_METHOD: Final = "isna"
+
+CONF_LAT_ADJ_METHOD: Final = "latitude_adjustment_method"
+LAT_ADJ_METHODS: list[str] = [
+    # PrayerTimesCalculator.LAT_ADJ_METHODS has values with space in them,
+    # which goes against translation_key requirement
+    value.replace(" ", "_")
+    for value in list(PrayerTimesCalculator.LAT_ADJ_METHODS)
```

Comment:
Can you please define all values statically here?

A dependency dump can introduce new values and it easily can happen that it will be forgotten to update the translations too.

Same also for `MIDNIGHT_MODES`, `SCHOOLS` and `CALC_METHODS`